Wikipedia/Code/TalkPageViewController.swift

```diff
@@ -276,6 +278,7 @@ class TalkPageViewController: ViewController {
         navVC.modalPresentationStyle = .pageSheet
         navVC.presentationController?.delegate = self
         present(navVC, animated: true, completion: nil)
+        presentTopicReplyOnboardingIfNecessary()
```

Comment:
I wonder if we should call this in the previous line's completion block, because you check for `presentedViewController` in this method. Is there a chance `presentedViewController` could not yet be assigned?

This is minor/optional, since it's working perfectly fine as-is with my testing.

Reply:
I recall having this same thought when originally working on this, and at the time left it afterwards just because it was working as expected. But I agree, it feels more _right_ having it in the completion block as if it's respecting some kind of order of operations. Updated!